acceptance_tests/cypress/integration/lotse_flow.spec.js

```diff
@@ -437,10 +431,7 @@ context('Acceptance tests', () => {
                 cy.get('select[id=person_b_religion]').select('ev')
                 cy.get(submitBtnSelector).click()
 
-                cy.get('label[for=person_b_has_disabilityyes]').click()
-                cy.get(submitBtnSelector).click()
-
-                cy.get('label[for=person_b_requests_pauschbetragyes]').click()
```

Comment:
If we aren't adding the disability test case in this PR (Which I honestly think we should) - can we add a sub-task to do that at one point? otherwise we might forget it.

Reply:
Added the tests